pytket/qir/conversion/api.py

```diff
@@ -115,3 +127,29 @@ def pytket_to_qir(
             return populated_module.module.ir()
         else:
             assert not "unsupported return type"  # type: ignore
+
+
+def _scratch_reg_resize_pass(max_size: int) -> CustomPass:
```

Comment:
This is coppied, I have set up a task to move this over to pytket to have only one place where this function can be maintained in one step: https://github.com/CQCL/tket/issues/975